### ISSUE TITLE
Clarify supported video formats

### ISSUE THREAD

**Issue by Beep6581** [alicevision/meshroom#1619]
The Meshroom manual is very vague regarding supported video formats:
https://meshroom-manual.readthedocs.io/en/latest/feature-documentation/core/supported-formats.html?highlight=video

> Video File formats
> .avi’, ‘.mov’, ‘.qt’, ‘.mkv’, ‘.webm’, ‘.mp4’, ‘.mpg’, ‘.mpeg’, ‘.m2v’, ‘.m4v’, ‘.wmv’, ‘.ogv’, ‘.ogg’, ‘.mxf’

Strictly speaking, those are container formats, not video formats. "mp4" could contain a wide variety of formats.

An exhaustive list of supported video formats would be unnecessary and unrealistic, but just a mention of the most common ones would help, e.g.: H.264, H.264, VP9.

Alternatively, a way to get Meshroom to list supported video formats in the current build.

**Comment by natowi** [comment]
Yes, "Supported video file extensions" might be a more precise title.

AliceVision includes ffmpeg in the build, but I´m not sure how it is utilized in the keyframeselection node.
https://github.com/alicevision/AliceVision/blob/4f06b6c56abb029128d1c0130b2b423a855b840b/CMakeLists.txt#L20

**Comment by Beep6581** [comment]
ffmpeg allows one to list the codecs and formats supported in a build using `ffmpeg -codecs` and `ffmpeg -formats`. If it's possible to query a meshroom exe or appimage for that info, that would be great, and should be documented.

The reason I asked for this is that while troubleshooting the other issues I opened regarding video import, it would have been great to know that I'm at least using a video file in a supported format.

**Comment by fabiencastan** [comment]
To do that, we need to find a way to do it from the opencv/ffmpeg api.
